.github/workflows/deploy.yml

```diff
@@ -28,12 +31,12 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install deps
+        run: pip install -U pip setuptools build
+      - name: Build sdist
+        run: python3 -m build
```

Comment:
It's good to see migrating to the new build tooling here. Just wanted to note it because it is a change from the previous workflow.

Reply:
Just released `0.7.1` https://pypi.org/project/qiskit-ibm-provider/, so I think this can be merged

I have the same PR open in `qiskit-ibm-runtime` - [#1148](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1148)
I was hoping to do a release there too today but it's not looking likely